pkg/local_object_storage/peapod/read.go

```diff
@@ -0,0 +1,89 @@
+package peapod
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util/slice"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+// Get reads data from the underlying database by the given object address.
+// Returns apistatus.ErrObjectNotFound if object is missing in the Peapod.
+func (x *Peapod) Get(addr oid.Address) ([]byte, error) {
+	var data []byte
+
+	err := x.bolt.View(func(tx *bbolt.Tx) error {
+		bktRoot := tx.Bucket(rootBucket)
+		if bktRoot == nil {
+			return fmt.Errorf("%w: missing root bucket", apistatus.ErrObjectNotFound)
+		}
+
+		val := bktRoot.Get(keyForObject(addr))
+		if val == nil {
+			return apistatus.ErrObjectNotFound
+		}
+
+		data = slice.Copy(val)
```

Comment:
i have already asked somewhere but did not get the answer: why do we keep importing that non-neo-go util from `neo-go`?

@roman-khimov

Reply:
Maybe because it's so nice and solves the problem?

Reply:
in next go upgrades we'll use `bytes.Clone`

Reply:
so iiuc we either duplicate this in `neofs-node` or write 2 lines in-place or ..?

Reply:
It's already written, can be used like that. It'd also be easier to identify and fix instances of it in #2251 if we don't have 100500 variations with local copies and open-coded versions.

Reply:
i really don't understand what exactly we discuss here, `neo-go` is our base lib along with std one, so, if no one minds, i'll leave this instruction as it is

Reply:
@carpawell so you don't mind using this feature?

Reply:
it is not a problem for me, i just do not get it. it won't stop me from pressing "Approve"

Reply:
> it won't stop me from pressing "Approve"

i mean when `bytes.Clone` (or even `slices.Clone`) will become available with newer Go version - u'll be still against using those utilities?

Reply:
It's not that you need `neo-go`, it's that you already have it. Introducing it as a dependency just for this function would be inappropriate, but we're already dependent on it for different reasons, so reusing some functions is just a natural thing to do. And yes, it'll all be changed with newer Go.